incubating/aws-s3/src/start.sh

```diff
@@ -0,0 +1,26 @@
+export AWS_ACCESS_KEY_ID=$(codefresh get context ${S3_INTEGRATION} --decrypt -o json |jq --raw-output '.spec.data.auth.jsonConfig.accessKeyId')
+export AWS_SECRET_ACCESS_KEY=$(codefresh get context ${S3_INTEGRATION} --decrypt -o json |jq --raw-output '.spec.data.auth.jsonConfig.secretAccessKey')
+export AWS_DEFAULT_REGION=$(echo "${REGION,,}")
+export AWS_PAGER=""
+
+cd $working_directory
+echo "Starting to upload files to S3"
+echo ""
+
+if [ -d $SOURCE ]; then
+  aws s3 cp $SOURCE s3://$BUCKET/$S3_PREFIX --recursive
```

Comment:
Should we test for some error?

Reply:
The AWS command will error out when the directory is missing

Reply:
The only thing is that I have seen when testing, it wont cause the step to fail. Not sure how to fix that.